src/python/pants/core/goals/run_integration_test.py

```diff
@@ -13,29 +12,34 @@
 
 
 @ensure_daemon
-def test_run_then_edit(use_pantsd: bool) -> None:
+def test_restartable(use_pantsd: bool) -> None:
     # These files must exist outside of a Pants `source_root` so that `coverage-py` doesn't try
     # to collect coverage metrics for them (as they are local to the chroot and coverage will
-    # error unable to find their source)
-    dirname = "not-a-source-root"
+    # error unable to find their source). We also need them to be in different locations for
```

Comment:
Took the opportunity to fix this test, which was writing to the same location in the sandbox in both invocations. Technically this is OK right now, but was a future footgun.